### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by miraydamartinez** [rba2124/pwp-capstones#5]
### Criteria 1: Valid Python Code
* _Score Level:_ 1 (Does Not Meet Expectations)
* _Comment(s):_ I had a lot of errors when trying to run this code. I wasn't sure which of the two files to look at, but this one had a '2' on it so I assumed it was the updated/final one. 
 
### Criteria 2: Implementation of Project Requirements
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ Code produces all the required functions and classes and calls them in an appropriate order.
 
### Criteria 3: Software Architecture
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ Code is separated into distinct classes and functions each which are invoked for their own purposes. 
 
### Criteria 4: Uses Python Language Features
* _Score Level:_ 3 (Meets Expectations)
* _Comment(s):_ Python language features are usually used, but there are 1-3 cases of a coded construction could have been accomplished by a Python builtin. See Issues #1 and #2 for examples.  There are also several lines where you return a print statement. 
 
### Overall Score: 12/16
Overall, nice job! You used Python to successfully create a book class with sub-classes for fiction and nonfiction.  It was also great to see that you added comments to certain functions! This is a great skill to have in programming because it helps others understand what your function is doing at a high level. Moving forward, I recommend reading more on return statements and how they work as well as experimenting with different Python built-in features (of which, there are many!).

**Comment by rba2124** [comment]
No, version 2 was never finished. TomeRater.py is the file I was given and
I returned it as the final project.
As for return statements, the project was somewhat simplistic, probably by
intention, in that it did not contemplate tie scores for such things as
most read book and most positive user. As a consequence, returning a single
object wouldn't produce the output I wanted. If I had returned a list or
dictionary, the print(Tome_Rater.method()) call statements would have
printed the { } and [ ] characters as well as quotation marks. In some
cases, I 'returned' an empty string, and in other cases, a print call with
no argument. You state that returning a print call isn't valid Python code,
yet Python let me do it. The code in TomeRater.py did not produce any
errors when I ran it. It includes all the tests that were in populate.py so
all you had to do was run TomeRater.py and it should have demonstrated that
everything worked. If you ran version 2, which was going to incorporate a
menu system, I decided not to finish it. Basically, I wanted to rewrite the
whole project, have output saved to files, use a menu system, but I don't
need a book tracking program, so I decided to invest in other directions.

While we are on the subject, you recommended reading more about return
statements, and experimenting with different built-in features. I note that
the course would have been better, more complete, if it had addressed
import statements, built-in libraries, and third party packages to a
greater extent.
